.eslint-dictionary.json

```diff
@@ -257,12 +265,14 @@
   "parens",
   "pathname",
   "pathnames",
+  "pbkdf2",
   "pbkdf2sync",
   "pbxproj",
   "pipelined",
   "pipenv",
   "pipfile",
   "pkill",
+  "placeindex",
```

Comment:
nit: it looks like usages of this misspelled word could be safely renamed in code.

Reply:
you're right, I looked into some of the location documentation and that's not a naming convention that has to be all lowercased from what I can tell. Will make the change.